.gitignore

```diff
@@ -66,8 +66,8 @@ tests/typings/**/dist
 /*.js
 allure-reporter.d.ts
 devtools.d.ts
-!babel.config.js
 !.eslintrc.js
+!jest.config.js
```

Comment:
Why is this done?

Reply:
I moved the Jest configurations from the package.json into a `jest.config.js`, given that we ignore all JS files in the root I had to ignore this new config file.